README.md

```diff
@@ -106,9 +107,12 @@ Also check out the sister project, [awesome-dynamic-analysis](https://github.com
 - [Deno](#deno)
 - [Gherkin](#gherkin)
 - [HTML](#html)
+- [JSON](#json)
 - [LaTeX](#latex)
+- [Makefiles](#make)
 - [Markdown](#markdown)
 - [Mobile](#mobile)
+- [Node.js](#nodejs)
```

Comment:
Should this category name `Node.js`? Or more `JavaScript (Serverside)`? I did not go for `Serverside JavaScript`, because when I would look for Node.js, I would search in near JavaScript)
I ask because Node.js is the runtime, like [Deno](https://deno.land/).

I have few experiences with it, but I assume tools for Node.js are also capable to be applied to Deno?